C/c4DocEnumerator.cc

```diff
@@ -40,27 +40,22 @@ class C4DocEnumerator::Impl : public RecordEnumerator, public fleece::InstanceCo
     Impl(C4Collection *collection,
          sequence_t since,
          const C4EnumeratorOptions &options)
-    :RecordEnumerator(asInternal(collection)->keyStore(), since, recordOptions(options))
-    ,_collection(asInternal(collection))
-    ,_options(options)
-    { }
+            : RecordEnumerator(asInternal(collection)->keyStore(), since, recordOptions(options)), _collection(asInternal(collection)),
+              _options(options) {}
 
     Impl(C4Collection *collection,
          const C4EnumeratorOptions &options)
-    :RecordEnumerator(asInternal(collection)->keyStore(), recordOptions(options))
-    ,_collection(asInternal(collection))
-    ,_options(options)
-    { }
+            : RecordEnumerator(asInternal(collection)->keyStore(), recordOptions(options)), _collection(asInternal(collection)),
+              _options(options) {}
 
-    static RecordEnumerator::Options recordOptions(const C4EnumeratorOptions &c4options)
-    {
+    static RecordEnumerator::Options recordOptions(const C4EnumeratorOptions &c4options) {
         RecordEnumerator::Options options;
         if (c4options.flags & kC4Descending)
             options.sortOption = kDescending;
         else if (c4options.flags & kC4Unsorted)
             options.sortOption = kUnsorted;
         options.includeDeleted = (c4options.flags & kC4IncludeDeleted) != 0;
-        options.onlyConflicts  = (c4options.flags & kC4IncludeNonConflicted) == 0;
+        options.onlyConflicts = (c4options.flags & kC4IncludeNonConflicted) == 0;
```

Comment:
The extra space was to line up this statement with the one above it since they are very similar; it exposes the similarity.

Reply:
Fixed with move to Clang-Format